README.md

```diff
@@ -14,6 +15,6 @@ Add you name and github profile here. Have fun with MERGE CONFLICTS (if you were
 - [Ayush Dhar Dubey](https://github.com/AyushDharDubey)
 - [Rayyan Khan](https://github.com/rkcoder101)
 - [Akshit Mandial](https://github.com/Akshit517)
+- [Akshit Mandial](https://github.com/Akshit517)
```

Comment:
@mansipanchal11 remove this line, this is duplicate

Reply:
removed duplicate lines